src/service-worker.js

```diff
@@ -0,0 +1,29 @@
+var cacheName = 'invoizr'
```

Comment:
Have a look to https://github.com/NekR/offline-plugin/blob/master/docs/update-strategies.md